docs/howto/start-agent-remotely.md

```diff
@@ -0,0 +1,73 @@
+# How to start the agent remotely
```

Comment:
Consider whether there is a way to more clearly distinguish the titles of these two guides.

One is how to do X with the registry, this is how to do X with...?

In future, we might think about whether the guides should be combined.

Reply:
As how-to guides I believe they must remain separate, as they show two distinct approaches to not depend on the end-user interaction with the application and their slightly distinct implications. They could be fused in a single explanation page, though, but then we'd loose the straightforwardness of a how-to guide.

So I think the ultimate solution is to have an explanation or reference page about the UP4W background agent being a startup task,  covering the reason for such design, its limitations and referencing those how-to guides. That would allow removing the explanatory verbiage as you suggested.

Reply:
Agreed!